airbyte-integrations/connectors/source-smartsheets/sample_files/catalog.json

```diff
@@ -0,0 +1,43 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "aws_s3_sample",
+        "supported_sync_modes": ["full_refresh"],
+        "source_defined_cursor": false,
+        "source_defined_primary_key": [["id"]],
+        "json_schema": {
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
```

Comment:
is the "json_schema" needed here? Now acceptance tests are skipping this property anyway